src/emulator/controller.ts

```diff
@@ -548,7 +548,7 @@ export async function startAll(options: Options, showUI: boolean = true): Promis
     }
 
     const rc = dbRulesConfig.normalizeRulesConfig(
-      dbRulesConfig.getRulesConfig(projectId, options),
+      dbRulesConfig.getRulesConfig(projectId!, options),
```

Comment:
You still need the `!` here (and below)? It looks like it's guaranteed to be a string now, so it shouldn't be needed.